bark_core/signatures.py

```diff
@@ -385,6 +387,17 @@ def get_authorized_pubkeys(
 
 def get_pubkey_from_git() -> Optional[Pubkey]:
     identifier = cmd("git", "config", "user.signingKey", check=False)[0]
+    gpg_format = cmd("git", "config", "gpg.format", check=False)[0]
+    # check if signingKey is ssh
+    if gpg_format and gpg_format == "ssh":
+        email = cmd("git", "config", "user.email", check=False)[0]
+        identifier = os.path.expanduser(identifier) # expand path if relative
+        with open(identifier, 'r') as file: # read public key
+            pubkey = file.read()
+        pubkey_tmp = f"{email} {pubkey}" # join email & pubkey to create correct format
```

Comment:
Instead of creating a temporary file with the correct format, we could return an `SshKey` object directly: `SshKey(f"{email} ".encode() + pubkey.read())`.

Reply:
This is an awesome idea! Will try it if i have some free time.